src/Core/Authorization/AuthorizationResolver.cs

```diff
@@ -739,23 +752,6 @@ public IEnumerable<string> GetRolesForEntity(string entityName)
         return EntityPermissionsMap[entityName].RoleToOperationMap.Keys;
     }
 
-    /// <summary>
```

Comment:
why remove this piece of code?

Reply:
unused with Visual Studio denoting "0 references"